src/new-apps/utils.ts

```diff
@@ -22,6 +23,27 @@ const getComingSoonMarketplaceApps = (
       "releaseDate" in app,
   ) as GetV2SaleorAppsResponse.ComingSoonSaleorApp[] | undefined;
 
+const getAppManifestUrl = (
+  marketplaceApp: GetV2SaleorAppsResponse.SaleorApp,
+) => {
+  if ("manifestUrl" in marketplaceApp) {
+    return marketplaceApp.manifestUrl;
+  }
+};
```

Comment:
```suggestion
const getAppManifestUrl = (
  marketplaceApp: GetV2SaleorAppsResponse.SaleorApp,
) => marketplaceApp?.manifestUrl
```

Reply:
Without `in` check it returns typecheck error, because SaleorApp is union of two types, for which one of them doesn't contain manifestUrl field and what is not covered by optional chaining:
```
Property 'manifestUrl' does not exist on type 'SaleorApp'.
  Property 'manifestUrl' does not exist on type 'ComingSoonSaleorApp'.
```